### DISCUSSION TITLE
Pls help configure debug in Intellij Idea

### DISCUSSION THREAD

**Discussion by GooDoCheck** [deepjavalibrary/djl#2745] [Show and tell]
Hello! I'm trying to use the library according to the guide, and unfortunately I can't check the results, because the console output is not verbose enough. Guide url: https://d2l.djl.ai/chapter_linear-networks/linear-regression-djl.html
I suggest setting up my IDE, so the guide is https://github.com/deepjavalibrary/djl/blob/master/docs/development/development_guideline.md,
but the ide still shows the error "Check the "Development Guideline"->Debug to enable array display". I am working in Intellij Idea 2021.2.4 ultimate. I work with maven builder, added 2 dependencies ai.djl:api:0.23.0 and ai.djl.mxnet:mxnet-engine:0.23.0.
Please respond, perhaps it is worth making adjustments to this manual)

**Comment by frankfliu** [comment]
You can configure your IntelliJ:

<img width="1235" alt="image" src="https://github.com/deepjavalibrary/djl/assets/24425551/fd6b515c-d995-4a94-9e26-bc0c52e7b60e">

You can adjust display NDArray range:
toDebugString(10000, 10, 10, 20, true)

**Reply by GooDoCheck** [reply]
Yeah! It's works in debug! Thank you! 
![image](https://github.com/deepjavalibrary/djl/assets/68955973/afa69ec1-4e04-4135-8f76-eadfd8a51aea)
But in the console it is still displayed with an exception when using the command "System.out.println(X.toString())" and "System.out.println(X)". Configuration "$MODULE_WORKING_DIR$" has already been applied... Where else to look?

**Reply by GooDoCheck** [reply]
It's work if use method X.toDebugString(true), how expression in debug...it's normal usecase?
